common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -61,21 +61,22 @@ object Mappings {
   val metadataMapping = nonDynamicObj(
     "dateTaken" -> dateFormat,
     "description" -> sStemmerAnalysedString,
```

Comment:
I suppose the question now is whether we want to analyse these other fields similar to the other fields so we could do cross field searching. Not really sure which side I fall on, worth asking though.